tests/conftest.py

```diff
@@ -96,3 +97,92 @@ def triples_shares_files(request, GaloisField, triples_polys, triples_files_pref
     t = request.param['t']
     write_polys(
         triples_files_prefix, GaloisField.modulus, N, t, triples_polys)
+
+
+@fixture
+def simple_router():
```

Comment:
NYC: I am a little confused, we have a `router.py` which seems to have a similar router and that is what I am using for runnign the programs. If we want to test that then do we need to duplicate this code here?

Reply:
Oh yes, I see. Are you using `router.py` in the tests or in the implementation code under `honeybadgermpc`?